### PR TITLE
allow all files as other_processed_files

### PR DESCRIPTION
**Pull request by andreacosolo**

### CODE REVIEW

File: src/encoded/tests/data/workbook-inserts/experiment_hi_c.json [hunk 1]
```diff
@@ -72,6 +72,9 @@
         "lab": "test-4dn-lab",
         "aliases": ["labsample:alias3"],
         "files": ["4DNFIO67APZ1", "4DNFIO67AZZ1"],
+        "other_processed_files": [{
+            "title": "Test FileFastq as opf",
+            "files": ["/files-fastq/4DNFIO67AZB1/"]}],
```
Comment (j1z0) [inline src/encoded/tests/data/workbook-inserts/experiment_hi_c.json:77]: Codacy found an issue: [A space is required before '}'.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6115226)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6115226&bid=20012688)

File: src/encoded/tests/data/workbook-inserts/experiment_hi_c.json (hunk 1 above)
Comment (sbreiff) [reply src/encoded/tests/data/workbook-inserts/experiment_hi_c.json:77]: To make the json consistent the convention would be to move the `}],` on its own line as the same level of indentation as "other_processed_files"

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6115226&bid=20012688)

File: src/encoded/tests/data/workbook-inserts/experiment_hi_c.json [hunk 2]
```diff
@@ -72,6 +72,10 @@
         "lab": "test-4dn-lab",
         "aliases": ["labsample:alias3"],
         "files": ["4DNFIO67APZ1", "4DNFIO67AZZ1"],
+        "other_processed_files": [{
+            "title": "Test FileFastq as opf",
+            "files": ["/files-fastq/4DNFIO67AZB1/"]
+            }],
```
Comment (j1z0) [inline src/encoded/tests/data/workbook-inserts/experiment_hi_c.json:78]: Codacy found an issue: [Expected indentation of 8 spaces but found 12.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6115226)

**Pull request merged**